protractor-conf.js

```diff
@@ -9,7 +9,7 @@ config.specs = [
 ];
 
 config.capabilities = {
-  browserName: 'chrome'
+  browserName: 'firefox'
```

Comment:
Ooops. I committed it by accident (I am using it locally, because of the problem I have with protractor+latest Chrome).